Pod/Classes/Plugin/Container/LoadingContainerPlugin.swift

```diff
@@ -26,32 +30,36 @@ public class LoadingContainerPlugin: UIContainerPlugin {
         translatesAutoresizingMaskIntoConstraints = false
         
         let widthConstraint = NSLayoutConstraint(item: self, attribute: .Width, relatedBy: .Equal,
-            toItem: nil, attribute: .NotAnAttribute, multiplier: 1, constant: spinningWheel.frame.width)
+                                                 toItem: nil, attribute: .NotAnAttribute, multiplier: 1,
```

Comment:
Just for style purposes, when this happen we should format with one of these options:
- If there are no blocks in the parameter list, we should avoid column indentation, only indenting by one level, like it was before this change.
```swift
let widthConstraint = NSLayoutConstraint(item: self, attribute: .Width, relatedBy: .Equal,
    toItem: nil, attribute: .NotAnAttribute, multiplier: 1,
    constant: spinningWheel!.frame.width)
```
- If there are blocks in the parameter list, or we want a column-style formatting for some reason, break the line on the first parameter (just after `(`) and then put a parameter per line, with a single indentation for each
```swift
let widthConstraint = NSLayoutConstraint(
    item: self, 
    attribute: .Width,
    relatedBy: .Equal,
    toItem: nil, 
    attribute: .NotAnAttribute,
    multiplier: 1,
    constant: spinningWheel!.frame.width)
```